WhaleSpotting/react-app/src/styles/Profile.scss

```
@@ -8,7 +8,8 @@
 }
 
 .profile-pane {
-    min-height: 100vh - $footerHeight - $headerHeight;
+    min-height: 100%;
+    box-sizing: border-box;
```

I'm not sure this works as you want, instead you can do 
```suggestion
    min-height: calc(100vh - #{$footerHeight} - #{$headerHeight});
```

---

Added